x-pack/plugins/timelines/common/alerts/constants.ts

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
Just making a note here because I thought these field names already lived in a package, but apparently they don't.

Follow up TODO: move these to shared package.

Reply:
Yep, will do!